templates/includes/sidebar.html

```diff
@@ -1,4 +1,18 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
```

Comment:
Check third point from task requirements:
If the driver clicks on username - the corresponding Driver detail page must open.